python/lsst/ip/diffim/getTemplate.py

```diff
@@ -146,11 +152,12 @@ def getOverlappingExposures(self, inputs):
            A struct with attributes:
 
            ``coaddExposures``
-               List of Coadd exposures that overlap the detector (`list`
-               [`lsst.afw.image.Exposure`]).
+               Dict of Coadd exposures that overlap the detector, indexed on
+               tract id (`dict` [`int`, `list` [`lsst.afw.image.Exposure`] ]).
            ``dataIds``
-               List of data IDs of the coadd exposures that overlap the
-               detector (`list` [`lsst.daf.butler.DataCoordinate`]).
+               Dict of data IDs of the coadd exposures that overlap the
+               detector, indexed on tract id
+               (`dict` [`int`, `list [`lsst.daf.butler.DataCoordinate`] ]).
```

Comment:
You're right, but I'm already changing the API slightly by making physical_filter a required arg. I don't want to rework `run` even more, since that would probably need an RFC.